src/test/java/core/basesyntax/dao/FruitDaoImplTest.java

```diff
@@ -0,0 +1,68 @@
+package core.basesyntax.dao;
+
+import static core.basesyntax.db.Storage.fruits;
+
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FruitDaoImplTest {
+    private static FruitDao fruitDao;
+
+    @BeforeClass
+    public static void beforeClass() {
+        fruitDao = new FruitDaoImpl();
+    }
+
+    @Test
+    public void addFruit_Ok() {
+        fruits.put("banana", 80);
+        fruits.put("apple", 75);
+        fruits.put("grape", 30);
+        fruits.put("mango", 16);
+        fruitDao.add("mango", 18);
+        fruitDao.add("pear", 16);
+        long actual = fruits.get("mango");
+        Assert.assertEquals(18, actual);
+        Assert.assertTrue(fruits.containsKey("pear"));
+    }
+
+    @Test
+    public void getAmountOfFruit_Ok() {
+        fruits.put("banana", 80);
+        fruits.put("apple", 75);
+        fruits.put("grape", 30);
+        fruits.put("mango", 16);
+        long actual = fruitDao.getFruitAmount("grape");
+        Assert.assertEquals(30, actual);
+    }
+
+    @Test (expected = RuntimeException.class)
+    public void getAmountOfAbsentFruit_NotOk() {
```

Comment:
```suggestion
    public void getFruitAmount_absentFruit_NotOk() {
```

There are a lot of ways to name your test methods. The main point is that they should have informative names and be consistent along with other developers in your team. For this task use such convention: `<methodUnderTest>_<state>_<expectedBehavior>`; For example, if we are testing the method register with a `null` user's age the test method name should be `register_nullAge_notOk`. `notOk` is because the test expects the register method to throw an exception.

fix naming in all places if needed